arsceneview/src/main/java/io/github/sceneview/ar/ARSceneView.kt

```diff
@@ -577,14 +572,6 @@ open class ARSceneView @JvmOverloads constructor(
         super.destroy()
     }
 
-    fun destroyARCore() {
```

Comment:
No reasons to place public function cuz it's internal behavior 
Same as all other lifecycle calls